community/modules/compute/schedmd-slurm-gcp-v5-node-group/README.md

```diff
@@ -0,0 +1,174 @@
+## Description
+
+> **_WARNING:_** This module is in active development and is therefore not
```

Comment:
Should we add a link to node-group in the module list (with experimental tag)?

Reply:
Done. Also noticed a lot of missing modules in community and added them to this PR.